test/integration/visitor_can_view_items_test.rb

```diff
@@ -17,30 +17,31 @@ class VisitorCanViewItemsTest < ActionDispatch::IntegrationTest
 
   test "visitor can view items by category" do
 
-    category_1 = OilType.create(name: "Lard")
-    category_2 = OilType.create(name: "Coconut Oil")
+    category_1 = Oil.new(name: "Lard")
+    category_2 = Oil.create(name: "Coconut Oil")
+    category_1.save
 
-    item_1 = Chip.create(name: "Slotachips", price: 20, description: "Super yummy", oil_type_id: category_1.id)
-    item_2 = Chip.create(name: "Trader Joe's BBQ", price: 15, description: "I'd trade slota for these!", oil_type_id: category_2.id)
-    item_3 = Chip.create(name: "Dang Coconut", price: 17, description: "Dang, these are good", oil_type_id: category_2.id)
-    item_4 = Chip.create(name: "Lard Yummies", price: 19, description: "Chock Full of Lard", oil_type_id: category_1.id)
+    item_1 = Chip.create(name: "Slotachips", price: 20, description: "Super yummy", oil_id: category_1.id)
+    item_2 = Chip.create(name: "Trader Joe's BBQ", price: 15, description: "I'd trade slota for these!", oil_id: category_2.id)
+    item_3 = Chip.create(name: "Dang Coconut", price: 17, description: "Dang, these are good", oil_id: category_2.id)
```

Comment:
Useless assignment to variable - `item_3`. Did you mean `item_1`?<br>Line is too long. [117/80]